packages/by-name/genpolicy-msft/package.nix

```diff
@@ -3,8 +3,9 @@
 
 { lib
 , fetchFromGitHub
-, fetchurl
+, fetchpatch
 , applyPatches
+, pkgs
```

Comment:
we never pass `pkgs` to a package expression. The idea of `callPackage` is to only pass the args needed.

Reply:
Thanks, changed.